internal/batches/service.go

```diff
@@ -679,26 +673,16 @@ func (svc *Service) resolveRepositorySearch(ctx context.Context, query string) (
 		}
 	}
 	if ok, err := svc.client.NewRequest(repositorySearchQuery, map[string]interface{}{
-		"query":       setDefaultQueryCount(query),
+		"query":       setDefaultQueryCount(query) + " select:repo",
```

Comment:
Here it should be as simple as adding `select:repo` (yuck for doing it like this in the string but OK). Seems like file matches are only used for deduping, in which case, all that related code can be removed.